packages/core/src/components/tooltip/tooltip.tsx

```diff
@@ -233,6 +259,8 @@ export class Tooltip {
         class={{
           visible: this.visible,
         }}
+        id={`ix-tooltip-${this.id}`}
```

Comment:
This can be overwritten by any application developer or iam wrong @nuke-ellington